rainier-core/src/main/scala/com/stripe/rainier/core/Multivariate.scala

```diff
@@ -0,0 +1,50 @@
+package com.stripe.rainier.core
+
+import com.stripe.rainier.compute._
+
+case class MVNormal(k: Int, cov: Covariance) extends Distribution[Seq[Double]] {
+ def translate(location: Vec[Real]): Distribution[Seq[Double]] = ???
+
+ def logDensity(seq: Seq[Seq[Double]]) =
```

Comment:
putting return types on everything is nice for readers on github as well as users of text editors without type hints.